activemq-broker/src/main/java/org/apache/activemq/broker/jmx/ManagementContext.java

```diff
@@ -690,10 +690,17 @@ public JmxRegistry(int port) throws RemoteException {
             super(port);
         }
 
-        @Override
+        private String getLookupName() {
+            if (getConnectorPath() == null || getConnectorPath().length() == 0) {
+                return LOOKUP_NAME;
+            }
 
+            return getConnectorPath().replaceAll("^/+", "").replaceAll("/+$", "");
```

Comment:
As getConnectorPath doesn't change, I guess it would make sense to only perform these regex's once (on setConnectorPath?)

Reply:
Done! Thanks for the review.